### ISSUE TITLE
Option to force max brightness

### ISSUE THREAD

**Issue by GamerzHell9137** [joel16/PSV-VSH-Menu#32]
The previous version of VHS Menu didn't lower the brightness in games but now it does once overclocked.
Any way we can get an option to stop lowering it?

**Comment by joel16** [comment]
Have you ever tried this?

**Comment by joel16** [comment]
Sorry it seemed like the link didn't paste. I meant to say have you ever tried this: https://github.com/ShroomKing/No-Lower-Brightness-Vita/releases

**Comment by Cade-H** [comment]
No-Lower-Brightness did not work for me, at least not with NoNpDrm games. Would be awesome to see his feature implemented into PSV-VSH.
